a10_octavia/controller/worker/flows/a10_pool_flows.py

```diff
@@ -114,6 +114,9 @@ def get_delete_pool_flow(self, members, health_mon, store, topology):
             requires=[constants.LOADBALANCER, constants.LISTENER, a10constants.VTHUNDER]))
         delete_pool_flow.add(persist_tasks.DeleteSessionPersistence(
             requires=[a10constants.VTHUNDER, constants.POOL]))
+        delete_pool_flow.add(a10_network_tasks.GetPoolsOnThunder(
```

Comment:
No need to add this task in the `get_delete_pool_flow()` as the task is meant for hardware thunder flows.
Instead add `a10constants.POOLS: None` to store of `delete_pool()` for vthunder flows.